### DISCUSSION TITLE
Any Apple Silicon Mx processor support?

### DISCUSSION THREAD

**Discussion by leonardotoro** [openai/whisper#2034] [Q&A]
As for title, there are any why to accelerate process using Apple Silicon GPU instead CPU only?

<img width="189" alt="image" src="https://github.com/openai/whisper/assets/44090318/5a4c8f90-a1e8-4297-a7e8-0ebc0bde8ede">

**Comment by glangford** [comment]
You can try `--device mps`, there has been mixed success with it as described here (search past discussions for "M1" or "mps")

* https://github.com/openai/whisper/discussions/51

Or there is this port of whisper which supports Apple silicon
* https://github.com/ggerganov/whisper.cpp

**Comment by stri8ed** [comment]
> As for title, there are any why to accelerate process using Apple Silicon GPU instead CPU only?
> 
> <img alt="image" width="189" src="https://private-user-images.githubusercontent.com/44090318/306241491-5a4c8f90-a1e8-4297-a7e8-0ebc0bde8ede.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAyODQ1NDUsIm5iZiI6MTcxMDI4NDI0NSwicGF0aCI6Ii80NDA5MDMxOC8zMDYyNDE0OTEtNWE0YzhmOTAtYTFlOC00Mjk3LWE3ZTgtMGViYzBiZGU4ZWRlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzEyVDIyNTcyNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWMzYjY1YjM1MTg2NmEzZTY4Y2VmYjM0NDQxNWZmZWNmNTA1YTZhZjJlMjc2OGI4OTExMjIyYzJlMWE1MWY4NjMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.cL3vhwYnp6kil5kr5i3HTixkNUwWVUzSCjpqX15_lX8">

How is the CPU only performance?
